indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java

```diff
@@ -176,7 +176,7 @@ public CommandLine parse( String[] args )
     @Override
     public String getPomPropertiesPath()
     {
-        return "META-INF/maven/org.sonatype.nexus/nexus-indexer/pom.properties";
+        return "META-INF/maven/org.apache.maven.indexer/indexer-core/pom.properties";
```

Comment:
java -jar indexer-cli was not working due this:
```
cstamas@Urnebes ~/Worx/apache-maven/maven-indexer  (master *)$ java -jar indexer-cli/target/indexer-cli-6.1.0-SNAPSHOT.jar -v
Unable determine version from JAR file.
```
This fixes it (as after code donation the coordinates changed, but seems this was neglected)

Reply:
There isn't any test that exercise that ?

Reply:
Sadly no :disappointed: